components/RichTextEditor.js

```diff
@@ -360,6 +380,18 @@ export default class RichTextEditor extends React.Component {
     }
   };
 
+  constructPreviewImageURL = async (service, id) => {
```

Comment:
@Betree : This was done in order to support Vimeo; note that in the commented out code we have to wait for the Vimeo api to respond. But since we aren't using that now it's fine to remove this; which I've done. 😄